exporter/splunkhecexporter/go.mod

```diff
@@ -137,3 +137,7 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatautil
 replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatatest => ../../pkg/pdatatest
 
 replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/golden => ../../pkg/golden
+
+replace go.opentelemetry.io/collector/config/configcompression => github.com/rnishtala-sumo/opentelemetry-collector/config/configcompression v0.0.0-20241204155200-78d83fd4ebca
```

Comment:
I would rather we merge the changes in core and use the latest main snapshot.

Reply:
Closing this PR in favor of - https://github.com/open-telemetry/opentelemetry-collector/pull/11805